### ISSUE TITLE
Handle new behavior of etlas deps

### ISSUE THREAD

**Issue by rahulmutt** [typelead/sbt-eta#6]
`etlas deps` no longer downloads dependencies automatically so a new task must be made that does so.

**Comment by y-taka-23** [comment]
As you mentioned in the Etlas-side, the older versions of Etlas are no longer supported. Therefore the priority of the issue is getting higher, I guess.

Are you already started to work on the topic? If you are too busy to treat it, I also have passion for dealing.

**Comment by Jyothsnasrinivas** [comment]
@y-taka-23 We have not yet started on a fix. The changes should be minor. `etlas deps` used to download any missing dependencies but it no longer does so. 

You must run `etlas  install --dependencies-only` before `etlas deps`. Moreover, `etlas deps` now takes a component argument (library/executable). You can see some sample usages by looking at the output of `etlas deps --help`.

Feel free to reach out if you are need help.

**Comment by y-taka-23** [comment]
Thanks @Jyothsnasrinivas, so I'm gonna try tackling this.

**Comment by Jyothsnasrinivas** [comment]
@y-taka-23 How's it going with sbt-eta?

**Comment by y-taka-23** [comment]
@Jyothsnasrinivas To be honest, I'm just starting to read the code, due to being occupied with my regular work... :(

**Comment by Jyothsnasrinivas** [comment]
@y-taka-23 No problem! Just checking incase you are blocked on something.

**Comment by rahulmutt** [comment]
This should be fixed with #10.
